cli/pcluster/config/mapping.py

```diff
@@ -0,0 +1,530 @@
+        },
+        "ephemeral_dir": {
+            "default": "/scratch",
+            "cfn": "EphemeralDir",
+        },
+        "encrypted_ephemeral": {
+            "default": False,
+            "type": BoolParam,
+            "cfn": "EncryptedEphemeral",
+        },
+        "custom_ami": {
+            "cfn": "CustomAMI",
+            "allowed_values": r"^ami-[0-9a-z]{8}$|^ami-[0-9a-z]{17}$",
+            "validator": ec2_ami_validator,
+        },
+        "pre_install": {
+            "cfn": "PreInstallScript",
+            # TODO add regex
+            "validator": url_validator,
+        },
+        "pre_install_args": {
+            "cfn": "PreInstallArgs",
+        },
+        "post_install": {
+            "cfn": "PostInstallScript",
+            # TODO add regex
+            "validator": url_validator,
+        },
+        "post_install_args": {
+            "cfn": "PostInstallArgs",
+        },
+        "extra_json": {
+            "type": JsonParam,
+            "cfn": "ExtraJson",
+        },
+        "additional_cfn_template": {
+            "cfn": "AdditionalCfnTemplate",
+            # TODO add regex
+            "validator": url_validator,
+        },
+        "tags": {
+            "type": JsonParam,
+        },
+        "custom_chef_cookbook": {
+            "cfn": "CustomChefCookbook",
+            # TODO add regex
+            "validator": url_validator,
+        },
+        "custom_awsbatch_template_url": {
+            "cfn": "CustomAWSBatchTemplateURL",
+            # TODO add regex
+            "validator": url_validator,
+        },
+        # Settings
+        "scaling_settings": {
+            "type": SettingsParam,
+            "default": "default",  # set a value to create always the internal structure for the scaling section
+            "referred_section": SCALING,
+        },
+        "vpc_settings": {
+            "type": SettingsParam,
+            "default": "default",  # set a value to create always the internal structure for the vpc section
+            "referred_section": VPC,
+        },
+        "ebs_settings": {
+            "type": EBSSettingsParam,
+            "referred_section": EBS,
+        },
+        "efs_settings": {
+            "type": SettingsParam,
+            "referred_section": EFS,
+        },
+        "raid_settings": {
+            "type": SettingsParam,
+            "referred_section": RAID,
+        },
+        "fsx_settings": {
+            "type": SettingsParam,
+            "referred_section": FSX,
+        },
+    }
+}
+
+GLOBAL = {
+    "type": Section,
+    "key": "global",
+    "params": {
+        "cluster_template": {
+            # TODO This could be a SettingsParam referring to a CLUSTER section
+            "default": "default",
+        },
+        "update_check": {
+            "type": BoolParam,
+            "default": True,
+        },
+        "sanity_check": {
+            "type": BoolParam,
+            "default": True,
```

Comment:
No it was already set to true (see examples/config and [documentation](https://docs.aws.amazon.com/parallelcluster/latest/ug/global.html#sanity-check))

Reply:
are you sure? look at the code: 
https://github.com/aws/aws-parallelcluster/blob/614287c112c35d39f259cc782b485692233a3af9/cli/pcluster/cfnconfig.py#L279

Reply:
When using the `easyconfig` the value [was initialized](https://github.com/aws/aws-parallelcluster/blob/614287c112c35d39f259cc782b485692233a3af9/cli/pcluster/configure/easyconfig.py#L176) to `true`.

There is inconsistency between documentation, examples and code... :| 
In any case, I'm for change the default to "true", to stop the cluster creation in case of errors, then the customers can disable it if needed.